admin/app/helpers/spree/admin/base_helper.rb

```
@@ -225,7 +225,7 @@ def preference_field(object, form, key, i18n_scope: '')
         case key
         when :currency
           content_tag(:div, form.label("preferred_#{key}", Spree.t(key, scope: i18n_scope)) +
-            form.currency_select("preferred_#{key}", current_store.supported_currencies.split(','), {}, { class: 'custom-select', disabled: current_store.supported_currencies.split(',').count == 1 }),
+            form.select("preferred_#{key}", current_store.supported_currencies.split(','), {}, { data: { controller: 'autocomplete-select' }, disabled: current_store.supported_currencies.split(',').count == 1 }),
```

RuboCop (1.80.2)

[convention] Line is too long. [212/150]

(Layout/LineLength)